docker/pact-stub/pact_stub.go

```diff
@@ -0,0 +1,154 @@
+package main
```

Comment:
What differences/benefits does this have over using the official version?

Reply:
It deals with cookies better. The "official" one compares the cookie headers as strings, whereas mine will match them on key-value, so is immune to Cypress adding it's own weird cookie or reordering the ones you set up.

Reply:
Cool, I'm not particularly 'gized about supporting our own mock server, but Pact has always been a game of trade-offs.